libs/NfcCoreLib/lib/Nci/phNciNfc.h

```diff
@@ -287,6 +287,24 @@ typedef enum phNciNfc_NfceeModes
    PH_NCINFC_NFCEEDISC_UNKNOWN
 }phNciNfc_NfceeModes_t;
 
+/**
+* \ingroup grp_nci_nfc
+* \brief NFCEE Power and Link Control Info contains power related identifier..
+*/
+typedef enum phNciNfc_PowerLinkMode
+{
+    /** NFCEE  NFCC decides identifier*/
+    phNciNfc_PLM_NfceeNfccDecides = 0x00,
```

Comment:
You probably don't need 'Nfcee' in the names.

Reply:
I have updated the pull request removing 'Nfcee' in the names.